datahub-web-react/src/app/entity/shared/tabs/Dataset/Validations/DatasetAssertionsList.tsx

```diff
@@ -162,10 +168,17 @@ export const DatasetAssertionsList = ({
                                 </Tag>
                             </Tooltip>
                         </div>
-                        <DatasetAssertionDescription
-                            description={description}
-                            assertionInfo={record.datasetAssertionInfo}
-                        />
+                        {record.datasetAssertionInfo ? (
+                            <DatasetAssertionDescription
+                                description={description}
+                                assertionInfo={record.datasetAssertionInfo}
+                            />
+                        ) : (
+                            <AssertionDescriptionContainer>
+                                {description ?? 'No description provided'}
+                            </AssertionDescriptionContainer>
```

Comment:
fallback for description if this is not a dataset assertion (temporary handling until we port assertions list UI over form saas)